### ISSUE TITLE
grasp in a real robot

### ISSUE THREAD

**Issue by William-feng** [atenpas/gpd#75]
As you wrote in the tutorial2"After selecting a grasp, you can get the information about where to place the robot hand from the GraspConfig message."  Can I  get the coordinates of the object in the robot coordinate system from the GraspConfig message.

**Comment by atenpas** [comment]
No, this package cannot tell you the coordinates of the object. It generates grasp poses independently of the object pose.
